modules/serverless-logic-upgrade-scaling-up-workflows-with-gitops-profile.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.34-to-1.35
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-scaling-up-workflows-with-gitops-profile_{context}"]
+= Scaling up workflows with the `gitOps` profile
+
+Workflows with the `gitops` profile that were previously scaled down must be scaled back up to continue operation.
+
+.Procedure
+
+* Scale the workflow back to `1` by executing the following command:
++
+[source,terminal]
+----
+$ oc patch SonataFlow <my-workflow> -n <target_namespace> --type=merge -p '{"spec": {"podTemplate": {"replicas": 1}}}'
+----
```

Comment:
I think we have two different procedures for doing sale to 0 and scale to 1.

In "Scaling down workflows with the gitOps profile" we set

```
spec:
  podTemplate:
    replicas: 0
```
and we do:

oc apply -f <my-workflow.yaml> -n <target_namespace>

In this rescale part, we propose doing oc patch xxxxxx..

Since both approaches are correct, I suggest to this in both sections:

1) we propose the **oc apply** xxxxx, in both sections.
2) and we say that, alternatively you can also do the **oc patch** , also in both sections.

does it make sense?

Reply:
I have updated both the sections